src/controller/postUrl.ts

```diff
@@ -0,0 +1,28 @@
+import express, { NextFunction } from "express";
+import { databaseModel } from "../model";
+import { logger } from "../services";
+import { generateCustomName, STATUS_CODES } from "../utils";
+import { handleError, sendResponse } from "../utils";
```

Comment:
You should combine these. Your eslint is not working. If it was, it should catch this.
Make sure you work on your eslint lint after this PR is merged and before you work on the next routes
```suggestion
import { generateCustomName, STATUS_CODES,  handleError, sendResponse } from "../utils";
```